### PR TITLE
chore(web): move correction-search execution timer to its own file 🕐 

### PR DESCRIPTION
**Pull request by jahorton**
@keymanapp-test-bot skip

### CODE REVIEW

File: common/web/lm-worker/src/main/correction/execution-timer.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
+/**
+ * This inner class is designed to help the algorithm detect its active execution time.
```
Comment (rc-swag) [inline common/web/lm-worker/src/main/correction/execution-timer.ts:2]: Is this comment still correct? This inner class...

**Review by rc-swag — APPROVED** [review]
Apart from the comment on the the comment.
lgtm

File: common/web/lm-worker/src/main/correction/execution-timer.ts (hunk 1 above)
Comment (jahorton) [reply common/web/lm-worker/src/main/correction/execution-timer.ts:2]: It's adjusted in the next PR (#11758).

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.59-alpha](https://keyman.com/downloads/releases/alpha/18.0.59)

**Pull request merged**